packages/wallet/manager/result.js

```diff
@@ -0,0 +1,17 @@
+import { cryptography } from '@liskhq/lisk-client';
+
+const verifyMessageValidator = (inputs) => {
```

Comment:
it needs to be moved to the custom hook

Reply:
but its not a custom hook

Reply:
then create one, also this function should move to the utiles, its not part of manager